repos/fdbt-site/src/constants/attributes.ts

```diff
@@ -18,7 +18,7 @@ export const STAGE_NAMES_ATTRIBUTE = 'fdbt-stage-names';
 
 export const DURATION_VALID_ATTRIBUTE = 'fdbt-duration-valid';
 
-export const PRODUCT_DETAILS_ATTRIBUTE = 'fdbt-product-details';
+export const CARNET_PRODUCT_DETAILS_ATTRIBUTE = 'fdbt-product-details';
```

Comment:
probably worth altering the string value too, to be fdbt-carnet-product-details.